### ISSUE TITLE
Outdated Information in PHP

### ISSUE THREAD

**Issue by davidshq** [10up/Engineering-Best-Practices#315]
The php md states that one should use a namespace or pseudo-namespace, and references the latter as being necessary due to namespaces not being support in WP's minimum PHP compatibility. However this has changed, WP now requires 5.6 as minimum (making namespaces available):
https://wordpress.org/about/requirements/

See also: https://wordpress.org/news/2019/04/minimum-php-version-update/

Not part of 10up, but curious whether this is something you guys will drop or if there is a reason to maintain the pseudo-namespacing?

**Comment by tlovett1** [comment]
Yep, you're right. @davidshq or anyone have time to work on a PR?
